index.html

```diff
@@ -12,7 +12,7 @@ <h2>
     <h3>We can't wait to share the Next Big Thing!</h3>
 
     <p>
-      Call for Participation is open! <a href="/cfp">Submit your proposal</a> today.
+      Call for Participation is closed! Decisions will be sent on or around January 24, 2023. Check the <a href="/cfp">CFP page</a> for more details.
```

Comment:
Is it worth keeping this, or should we just remove this block entirely?

Reply:
I'm open to either. I kept it open to put timelines there for anyone who was wondering. If you have a better suggestion, I'm all ears.